screens/WelcomeScreen.tsx

```diff
@@ -5,35 +5,29 @@ import MaskedView from '@react-native-masked-view/masked-view'
 import EStyleSheet from 'react-native-extended-stylesheet';
 
 function WelcomeScreen({ navigation }) {
-    const backgroundCircle = require('../assets/background_blob.png');
-
     return (
         <View style={styles.container}>
-            <Image source={backgroundCircle} style={styles.ellipse1} blurRadius={70}></Image>
-            <Image source={backgroundCircle} style={styles.ellipse2} blurRadius={70}></Image>
-            <Image source={backgroundCircle} style={styles.ellipse3} blurRadius={70}></Image>
-            <Image source={backgroundCircle} style={styles.ellipse4} blurRadius={70}></Image>
-            <Image source={backgroundCircle} style={styles.ellipse5} blurRadius={70}></Image>
-            <Image source={backgroundCircle} style={styles.ellipse6} blurRadius={70}></Image>
-            <MaskedView style={styles.titleContainer}
-                maskElement={
-                    <View
-                        style={{
-                            backgroundColor: 'transparent',
-                            flex: 1,
-                            justifyContent: 'center',
-                            alignItems: 'center',
-                        }}
-                    >
-                        <Text style={styles.titleText}>see the good <Text style={{ color: 'transparent' }}>🌱</Text></Text>
-                    </View>
-                }
-            >
-                <LinearGradient colors={['#FF000F', '#A40811']} start={{ x: 0, y: -0.3 }} end={{ x: 0, y: 2 }} locations={[0, 0.7]}>
-                    <Text style={[styles.titleText, { opacity: 0 }]}>see the good <Text style={{ color: 'transparent' }}>🌱</Text></Text>
-                </LinearGradient>
-            </MaskedView>
-            <Image source={require('../assets/seedling.png')} style={styles.seedling} />
+            <View style={styles.titleContainer}>
+                <MaskedView
+                    maskElement={
+                        <View
+                            style={{
+                                backgroundColor: 'transparent',
+                                flex: 1,
+                                justifyContent: 'center',
+                                alignItems: 'center',
+                            }}
+                        >
+                            <Text style={styles.titleText}>see the good </Text>
+                        </View>
+                    }
+                >
+                    <LinearGradient colors={['#FF000F', '#A40811']} start={{ x: 0, y: -0.3 }} end={{ x: 0, y: 2 }} locations={[0, 0.7]}>
+                        <Text style={[styles.titleText, { opacity: 0 }]}>see the good </Text>
+                    </LinearGradient>
+                </MaskedView>
+                <Image source={require('../assets/gift.png')} style={styles.gift} />
```

Comment:
Gift image and title text are put into a View horizontally, so it is properly spaced from the title text with a space character at the end of the title text.